### ISSUE TITLE
Save Image on iOS crashes 100% of the time

### ISSUE THREAD

**Issue by computer2467** [mamaral/xkcd-Open-Source#86]
REPRO:
Go to any comic, tap the share button, tap the save image button.

SPECS:
iPhone X, iOS v12.3.1
App version v2.5

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Kamik423** [comment]
I had an issue in an app of mine where something similar would happen, because the app has get permission to save an image. [StackOverflow](https://stackoverflow.com/questions/39743492/ios-10-app-crashes-when-trying-to-save-image-to-photo-library).

This might be related. In my case the app crashed immediately, xkcd:OS however freezes.

Looking through the code: this seems to be in there already:

https://github.com/mamaral/xkcd-Open-Source/blob/c23cb9db60a714a3c7d9ca3899d5a610868a24be/Source/Misc/Supporting%20Files/Info.plist#L72-L73

However as of iOS 11 the key `NSPhotoLibraryAddUsageDescription` seems to be more applicable.

# TL;DR:

I thought this was it, but it is not. So you can cross this off your list.
